client/src/pages/SubmissionDetails.vue

```diff
@@ -65,6 +65,18 @@
         mutable
       />
     </div>
+    <section class="q-pa-lg">
+      <h3>Activity</h3>
+      <p v-if="submission.audits.length == 0">No Activity</p>
+      <ul>
+        <li
+          v-for="audit in submission.audits.slice().reverse()"
+          :key="audit.id"
+        >
+          <submission-audit :audit="audit"></submission-audit>
```

Comment:
```suggestion
          <submission-audit :audit="audit" />
```
I'm actually not sure why this isn't showing a style error. https://vuejs.org/style-guide/rules-strongly-recommended.html#self-closing-components